zipkin-core/src/main/java/com/smoketurner/dropwizard/zipkin/KafkaZipkinFactory.java

```diff
@@ -69,17 +70,17 @@ public void setTopic(String topic) {
      */
     @Override
     public Brave build(@Nonnull final Environment environment) {
-        final SpanCollectorMetricsHandler metricsHandler = new DropwizardSpanCollectorMetricsHandler(
+        final ReporterMetrics metricsHandler = new DropwizardReporterMetrics(
                 environment.metrics());
 
-        final KafkaSpanCollector.Config kafkaConfig = KafkaSpanCollector.Config
-                .builder(bootstrapServers).topic(topic).build();
-        final SpanCollector spanCollector = KafkaSpanCollector
-                .create(kafkaConfig, metricsHandler);
+        final KafkaSender sender = KafkaSender.builder()
+                .bootstrapServers(bootstrapServers).topic(topic).build();
+        final AsyncReporter<Span> reporter = AsyncReporter.builder(sender)
```

Comment:
I'll set up a ManagedSender instance so Dropwizard can shut it down when Jetty is stopped.  http://www.dropwizard.io/1.0.2/docs/manual/core.html#managed-objects

Thanks!